src/main/java/com/ensao/gi5/lint/wrapper/ClassAttributesWrapper.java

```diff
@@ -0,0 +1,34 @@
+package com.ensao.gi5.lint.wrapper;
+
+import com.ensao.gi5.lint.util.Utils;
+import com.github.javaparser.ast.body.EnumConstantDeclaration;
+import com.github.javaparser.ast.body.FieldDeclaration;
+
+public class ClassAttributesWrapper {
+
+    private String classAttribureFQN;
+    private String classAttribure;
+    private int line;
+
+    public ClassAttributesWrapper(FieldDeclaration fieldDeclaration){
+        this.classAttribureFQN = fieldDeclaration.getVariable(0).getNameAsString();
```

Comment:
getVariable(0) has to be generalized, a field may have more than one declared variable, exemple 
```java 
private String foo, bar;
```